src/Workspaces/Core/Portable/Options/OptionsExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.CodeStyle;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    internal static class OptionsExtensions
+    {
+        public static Option<CodeStyleOption<T>> ToPublicOption<T>(this Option2<CodeStyleOption2<T>> option)
```

Comment:
Extension methods for public types such as `CodeStyleOptions`, `FormattingOptions` etc. to convert internal options types into public types that were already exposed by these.